gradle.properties

```diff
@@ -11,7 +11,7 @@ android.suppressUnsupportedOptionWarnings=android.suppressUnsupportedOptionWarni
 kotlin.code.style=official
 kotlin.mpp.stability.nowarn=true
 GROUP=dev.yorkie
-VERSION_NAME=0.4.25-rc
+VERSION_NAME=0.4.27-rc
```

Comment:
I set the RC version because even after the merge this PR, there are still specs to discuss. 
https://github.com/yorkie-team/yorkie-js-sdk/pull/865